kiali/kiali
# Display full yaml from istio configmap

### Describe the change

Display full yaml from istio configmap in the Istio Mesh page

### Steps to test the PR

- Click in Mesh page
- Click in the Istio icon
- See istio config map

![image](https://github.com/user-attachments/assets/39de9c2f-96cb-43ac-b1e8-a021d48c77fa)


### Automation testing
Updated unit tests

### Issue reference

Fixes #7459
